libpod/container_internal.go

```diff
@@ -202,18 +218,17 @@ func (c *Container) setupStorage(ctx context.Context) error {
 		return errors.Wrapf(ErrCtrStateInvalid, "container %s must be in Configured state to have storage set up", c.ID())
 	}
 
-	// Need both an image ID and image name, plus a bool telling us whether to use the image configuration
-	if c.config.RootfsImageID == "" || c.config.RootfsImageName == "" {
```

Comment:
We want to preserve this test here - ensure that there is either Rootfs is not "", or RootfsImageID and RootfsImageName are not "", to verify that we have a good image to work with if one is given, and we don't get a rootfs and an image at the same time.